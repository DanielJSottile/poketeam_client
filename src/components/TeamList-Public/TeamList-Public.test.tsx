import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
//import { render } from '@testing-library/react';
import TeamListPublic from "./TeamList-Public";
import { shallow, mount, render } from "enzyme";
import toJson from "enzyme-to-json";

describe("<TeamsListPublic/>", (): void => {
  it("renders without crashing", (): void => {
    const div: HTMLDivElement = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <TeamListPublic />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it("snapshot", (): void => {
    const wrapper = shallow(<TeamListPublic />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
