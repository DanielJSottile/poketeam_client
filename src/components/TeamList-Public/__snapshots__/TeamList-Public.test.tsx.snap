// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamsListPublic/> snapshot 1`] = `
<Fragment>
  <div
    className="pokeball-div"
  >
    <PokeballLoader />
    <LoadingBlack />
    <h3
      className="hint"
    >
      (Hint: There May Be No Teams)
    </h3>
  </div>
</Fragment>
`;
